src/Cfonb120Reader.php

```diff
@@ -25,6 +25,10 @@ class Cfonb120Reader extends AbstractReader
 {
     /** @var Statement[] */
     private $statements = [];
+    private const IGNORE_LINES = [
```

Comment:
I'm note sure to give this package the responsibility of ignoring some lines. Is it really a bug for you?

Reply:
Yeah if this line are not ignored this will throw an exception.
But i understand that this implementation of ignored lines is very naive.
Like lineParse i could add a similar mechanism "LineParserIgnore" with same method `supports(string $content): bool`

WDYT ?

Reply:
I've made some tests with your file.
I think your problem could be fixed by setting the `_unused_4 ` to `[self::ALL, 50]` instead of `[self::BLANK, 50]` in the `Line01Parser` class.

Reply:
with this diff


```diff
diff --git a/src/Cfonb120Reader.php b/src/Cfonb120Reader.php
index 8e3b04d..e97d89e 100644
--- a/src/Cfonb120Reader.php
+++ b/src/Cfonb120Reader.php
@@ -55,12 +55,12 @@ class Cfonb120Reader extends AbstractReader
         foreach ($lines as $line) {
             if (empty($line)) {
                 continue;
-            }
+            }/*
             foreach (self::IGNORE_LINES as $toIgnore) {
                 if (strpos(strtolower($line), strtolower($toIgnore))) {
                     continue 2;
                 }
-            }
+            }*/
 
             foreach ($this->parsers as $parser) {
                 if (!$parser->supports($line)) {
diff --git a/src/Parser/Cfonb120/Line01Parser.php b/src/Parser/Cfonb120/Line01Parser.php
index f2246c6..0dccbcb 100644
--- a/src/Parser/Cfonb120/Line01Parser.php
+++ b/src/Parser/Cfonb120/Line01Parser.php
@@ -28,7 +28,7 @@ class Line01Parser extends AbstractCfonb120Parser
             'account_nb' => [self::ALPHANUMERIC, 11],
             '_unused_3' => [self::BLANK, 2],
             'date' => [self::NUMERIC, 6],
-            '_unused_4' => [self::BLANK, 50],
+            '_unused_4' => [self::ALL, 50],
             'amount' => [self::AMOUNT, 14],
             '_unused_5' => [self::ALL, 16],
         ]);

```

test fail 

```console
) Siarhi\Cfonb\Tests\Cfonb120ReaderTest::testComplexTest
Silarhi\Cfonb\Exceptions\ParseException: Unable to find a parser for the line :
"0710278    02204EUR2 00020427603  240420        ------------------------------            0000005843530B                 "
``

Reply:
I see 121 chars on this line :-) I think you've added an extra whitespace by error during the copy/paste.

Reply:
done and working :+1: good catch